src/llvm.cpp

```diff
@@ -1732,16 +1749,26 @@ void emit_decl_var(IRContext *self, Decl *decl) {
                 return;
             }
         }
-        Type *type = llvm_type(self, sym->type);
+		
+        Type *type = llvm_type(self, sym->type, true);
```

Comment:
It seems like I need the `true` here for global variables but I'm not exactly sure if I need it for all cases. Thoughts?